daemon.lua

```diff
@@ -47,9 +47,16 @@ local function daemon()
 		end
 		local frame = inbox.reply:receive()
 		if frame then
+			local what, frame = frame:sub(1, 5), frame:sub(6)
```

Comment:
I would use pattern matching (using a separator) here to extract the function name and the packet, instead of counting chars, and then `reply[method](packet)`